.github/workflows/zap-vs-vulnerableapp.yml

```diff
@@ -0,0 +1,17 @@
+name: ZAP vs VulnerableApp
+
+on:
+  workflow_dispatch:
+
+jobs:
+  scan:
+    runs-on: [ubuntu-latest]
+    steps:
+    - name: Start vulnerableApp
+      run: docker run --rm -d -p 9090:8080 --name=owasp-vulnerableapp sasanlabs/owasp-vulnerableapp:latest
+    - name: Start ZAP
+      run: docker run --rm -d -p 8090:8090 --network host owasp/zap2docker-weekly zap-x.sh -daemon -host localhost -port 8090 -addonupdate -addoninstall domxss -config "api.addrs.addr.name=.*" -config api.addrs.addr.regex=true -config api.disablekey=true &
```

Comment:
@psiinon i was looking to add the step of firing requests after spidering using the information provided by `scanner` endpoint for more better result comparison. Where can we do that ?